### PR TITLE
Require stable fsi/admin-bundle 3.0 branch

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

**Comment by szymach** [comment]
I have added all Symfony components that were used in any of the classes of the bundle, seems to help.

File: composer.json [hunk 1]
```diff
@@ -15,29 +15,37 @@
         }
     ],
     "require": {
+        "fsi/admin-bundle" : "~2.0|~3.0",
         "php": ">=7.1",
-        "symfony/framework-bundle" : "^2.3|^3.0|^4.0",
-        "fsi/admin-bundle" : "^2.0|^3.0@dev"
+        "symfony/config" : "~2.3,>=2.3.12|~3.0|~4.0",
```
Comment (szymach) [inline composer.json]: The precise versions for branches `2.3` were copied from requirements found for `symfony/framework-bundle`.

**Comment by szymach** [comment]
@fsi-open

File: composer.json (hunk 1 above)
Comment (chives) [reply composer.json]: `^2.3.12|^3.0|^4.0`

**Comment by szymach** [comment]
@fsi-open

**Pull request merged**